src/types/markers.js

```diff
@@ -75,11 +74,11 @@ type GCSliceData_Shared = {
   trigger_amount?: number,
   trigger_threshold?: number,
 
-  // The number of page faults that occured during the slice.
-  page_faults: number,
+  // The number of page faults that occured during the slice.  If missing
+  // there were 0 page faults.
+  page_faults?: number,
 
   start_timestamp: Seconds,
-  end_timestamp: Seconds,
```

Comment:
Note we could have some older profiles with `end_timestamp` and `when` so it's not technically correct. But because we never used these fields, I'd say it's fine.

Reply:
True, I don't think I thought of that at the time.  However I would still prefer to remove it to attempt to keep this type clean-ish.